onadata/libs/templates/account_lockout/lockout_end.txt

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+{% blocktrans with username=username %}
+Hi {{ username }},
+
+You are no longer locked out of Ona, you can now log in to your account.
+Thank you for choosing Ona!
+Please contact us with any questions, we're always happy to help.
```

Comment:
How will they contact us - are we providing a contact address or directions to how to reach us? (similar comment below) Otherwise, I am fine with this email

Reply:
I have included `support@ona.io` in the email. Have a look and suggest any improvements on the email.